base-install.nix

```
@@ -78,6 +93,10 @@ in
       extraUsers.root.hashedPassword = mkBaseDefault "$y$j9T$LEwPZpyLzb3CKDBEtAi.w1$Uxok0mk4i5AWJ0zbPaqfY6T7Bw5nNYteu69yxqD7Mg/";
     };
 
+    environment.systemPackages = with pkgs; [
+      lp-tool
```

...once you've added `wind-tunnel` as an input in the `flake.nix` then you can change this to:

```suggestion
      inputs.wind-tunnel.packages.x86_64-linux.lp-tool
```